### PR TITLE
create react routers. start loading spinner.

### PR DESCRIPTION
**Pull request by TheunisK**

### CODE REVIEW

File: src/client/components/Header.jsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
         <header className="homeHeader">
             <div className="headerTitle"></div>
             <div className="headerNav">
             <ul>
-                <li>Profile</li>
-                <li>Login</li>
-                <li>Ratings</li>
+                <li><Link to="/home">Home</Link></li>
+                <li><Link to="/profile">Profile</Link></li>
+                <li><Link to="/ratings">Ratings</Link></li>
```
Comment (Can-Tina) [inline src/client/components/Header.jsx:11]: Nice links

File: src/client/components/Login.jsx [hunk 2]
```diff
@@ -77,7 +83,7 @@ const Login = () => {
                     >Login</button>
                     <label className="registerLabel">
                         Don't have an account?
-                        <a className="registerLink" id="registerLink"> Register Here</a>
+                        <Link to="/registration" className="registerLink" id="registerLink"> Register Here</Link>
```
Comment (Can-Tina) [inline src/client/components/Login.jsx:86]: Nice :)

File: src/client/components/Pokedex.jsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ const Pokedex = () => {
     }
 
     useEffect(() => {
-        getPokemons()
+        getPokemons();
```
Comment (Can-Tina) [inline src/client/components/Pokedex.jsx:25]: can't forget the semicolon

File: src/client/components/Registration.jsx [hunk 4]
```diff
@@ -58,7 +61,7 @@ const Registration = () => {
                         Email
                     </label>
                     <input 
-                        type="text" 
+                        type="email"
```
Comment (Can-Tina) [inline src/client/components/Registration.jsx:64]: Ahh good, we needed to sort that out

File: src/client/components/Header.jsx (hunk 1 above)
Comment (glowkeeper) [reply src/client/components/Header.jsx:11]: looks like frontend routing is coming to life...

**Comment by glowkeeper** [comment]
Approved

**Review by glowkeeper — APPROVED** [review]
Nice work

**Pull request merged**